src/gamemodes/amongus/gamemode/sv_net.moon

```diff
@@ -538,3 +540,27 @@ GM.SetOnAutoPilot = (newState) =>
 -- @return You guessed it again.
 GM.SetFullyInitializedPlayerCount = (newCount) =>
 	SetGlobalInt "NMW AU FullyInitializedPlayers", newCount
+
+GM.Net_SyncGameSettings = (ply) =>
+	for _, cvar in ipairs GAMEMODE.replicatedWritableCvars
+		net.Start "AU ChangeCvar"
+		net.WriteString GAMEMODE.ConVars[cvar]\GetName!
+		net.WriteString GAMEMODE.ConVars[cvar]\GetString!
+		net.Send ply
+
+skipSync = {}
+for _, cvar in ipairs GAMEMODE.replicatedWritableCvars
+	cvars.AddChangeCallback GAMEMODE.ConVars[cvar]\GetName!, ((cvar, oldValue, newValue) ->
+		if skipSync[cvar]
+			skipSync[cvar] = false
+		else
+			net.Start "AU ChangeCvar"
+			net.WriteString cvar
+			net.WriteString newValue
+			net.Broadcast!
+	), "SendToClient"
+
+net.Receive "AU ChangeCvar", (len, ply) ->
+	cvar = net.ReadString!
+	skipSync[cvar] = true
+	RunConsoleCommand cvar, net.ReadString!
```

Comment:
This would allow for users with a clientside Lua bypass to send arbitrary console commands to the server (albeit with only one parameter), no? The CAMI permissions check probably ought to happen here rather than in `cl_net.moon`.

Reply:
Youre right but I would keep the check in `cl_net.moon` and just add another one here

Reply:
That's fine. I just want to make sure that a) only authorized users can trigger that netmessage, and b) only convars in the replicatedWriteable list can actually be manipulated.

Reply:
I cant test it right now but it should be fixed